### PR TITLE
[Reviewer AMC] Default hss_reregistration_time based on reg_max_expires

### PR DESCRIPTION
**Pull request by steelyvoid**
Related PR for https://github.com/Metaswitch/homestead/pull/295

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -1349,7 +1349,11 @@ int main(int argc, char* argv[])
   opt.external_icscf_uri = "";
   opt.auth_enabled = PJ_FALSE;
   opt.enum_suffix = ".e164.arpa";
+
+  // If changing this default for reg_max_expires, note that a corresponding
+  // change is required to debian/homestead.init.d in the homestead repository
```
Comment (bossmc) [inline src/main.cpp]: What is the associated change?  It's not immediately clear.

**Comment by bossmc** [comment]
One comment.

**Comment by steelyvoid** [comment]
Hi Andy. Is that wording any clearer?

**Comment by bossmc** [comment]
Fine.

**Pull request merged**